docs/get-started/installation/custom-hardware/installation/script.md

```diff
@@ -34,10 +34,10 @@ To finalize the installation once it has finished completing, reboot your machin
 
 Once the installation process has finished, DAppNode will try to automatically prepare the first access to your DAppNode on the next order:
 
-1. [Local Proxy](../../../../user-guide/ui/access/local-proxy)
-2. [Wifi](../../../../user-guide/ui/access/wifi)
-3. [Wireguard](../../../../user-guide/ui/access/vpn#wireguard)
-4. [OpenVPN](../../../../user-guide/ui/access/vpn#openvpn)
+1. [Local Proxy](https://docs.dappnode.io/user-guide/ui/access/local-proxy)
+2. [Wifi](https://docs.dappnode.io/user-guide/ui/access/wifi)
+3. [Wireguard](https://docs.dappnode.io/user-guide/ui/access/vpn#wireguard)
+4. [OpenVPN](https://docs.dappnode.io/user-guide/ui/access/vpn#openvpn)
```

Comment:
Please revert this changes ASAP, don't use absolute links you MUST use relative links but the correct relative link

Reply:
You can add tooling in CI to ensure all links are correct, google it

Reply:
I know its a solution, the problem happens not because the relative path is wrong:
- if you click on the link it works but if you have clicked on some of the title that is on the right side(clicking on this title change your path and add #title) the links do not work, this issue is not always happening but yes in this case. I asked for help in docusaurus but they have not answered me yet.
- It would be a good idea to add a tool to ensure to all links are correct. 
I will revert the changes and add this tools.